api/nativeMessaging/background.js

```diff
@@ -0,0 +1,8 @@
+chrome.runtime.onInstalled.addListener((e) => {
+  console.log(e, chrome.runtime.getManifest());
+});
+
+chrome.action.onClicked.addListener(() => 
+  chrome.runtime.sendNativeMessage('com.google.chrome.example.ping.pong'
+  , {message:"ping"}, (nativeMessage) => console.log(nativeMessage))
+);
```

Comment:
```suggestion
  chrome.runtime.sendNativeMessage(
    'com.google.chrome.example.ping.pong',
    {message:"ping"},
    (nativeMessage) => console.log(nativeMessage)));
```

Reformat in accordance with [4.6.4 Function arguments](https://google.github.io/styleguide/jsguide.html#formatting-function-arguments). @samthor, is this correct?